keyboards/yd60mq/README.md

```diff
@@ -0,0 +1,18 @@
+YD60MQ
```

Comment:
Please also rename this file to lowercase, to avoid case sensitivity problems (Linux/macOS are case sensitive but Windows isn't).